autofix/rules/empty-line-between-blocks.js

```diff
@@ -7,6 +7,18 @@ function emptyLineBetweenBlocks(tree) {
     if (node.is('ruleset') && index > 0) {
       const prev = parent.get(index - 1);
 
+      // fix only if any of leading elements is anything than space
```

Comment:
in general I think this could be written in a for loop which would be more readable then `while(true) with break and return` but I do not insist - if it does the job 👍